testsuites/basic/lsvmbus.py

```diff
@@ -0,0 +1,112 @@
+import math
+from pathlib import Path
+from typing import Optional
+
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.environment import EnvironmentStatus
+from lisa.features import SerialConsole
+from lisa.testsuite import simple_requirement
+from lisa.tools import Lscpu, Lsvmbus
+from lisa.util import LisaException
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite uses to check correctness of vmbus channel
+    """,
+    tags=[],
+)
+class LsVmBus(TestSuite):
+    TIME_OUT = 300
+    __Tokens = [
+        "Operating system shutdown",
+        "Time Synchronization",
+        "Heartbeat",
+        "Data Exchange",
+        "mouse",
+        "keyboard",
+        "Synthetic network adapter",
+        "Synthetic SCSI Controller",
+    ]
+
+    @TestCaseMetadata(
+        description="""
+        This test is to check channel count of each netvsc and storvsc SCSI device.
+        When count is unexpected, then test fail.
+        """,
+        priority=0,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole],
+        ),
+    )
+    def lsvmbus_test(self, case_name: str) -> None:
+        node = self.environment.default_node
+        case_path: Optional[Path] = None
+
+        is_ready, tcp_error_code = wait_tcp_port_ready(
+            node.public_address, node.public_port, log=self.log, timeout=self.TIME_OUT
+        )
+        if not is_ready:
```

Comment:
It doesn't need in vmbus test case. It should be handled by smoke test. If the image may not be ready, we need to capture the serial console log. It should be in deployment code.